Eevee.Sleep.Bot/Utils/DiscordMessageMaker.cs

```diff
@@ -171,4 +171,22 @@ public static class DiscordMessageMaker {
             .WithCurrentTimestamp()
             .Build();
     }
+
+    public static MessageComponent MakeRoleSelectButton(ulong[] hasRoles, IEnumerable<SocketRole> guildRoles) {
+        var builder = new ComponentBuilder();
+
+        foreach (var role in hasRoles) {
+            builder.WithButton(
+                label: guildRoles.Where(x => x.Id == role).Select(x => x.Name).First(),
+                customId: ButtonInteractionInfoSerializer.Serialize(
+                    new ButtonInteractionInfo() {
+                        ButtonId = ButtonId.RoleChanger,
+                        CustomId = role.ToString()
```

Comment:
`ButtonInteractionInfo()` should be able to take role ID as `ulong` then convert it on init so no need to convert role ID from `ulong` to `string`.